client/index.html

```diff
@@ -122,75 +122,81 @@ <h1 class="title">Sign in to The Lounge</h1>
 						<div class="row">
 							<div class="col-sm-12">
 								<h1 class="title">
-									<%= public ? "The Lounge - " : "" %>
+									{{#if public}}The Lounge - {{/if}}
 									Connect
-									<%= !displayNetwork && lockNetwork ? "to " + defaults.name : "" %>
+									{{#unless displayNetwork}}
+										{{#if lockNetwork}}
+										to {{defaults.name}}
+										{{/if}}
+									{{/unless}}
 								</h1>
 							</div>
-							<div <%= typeof(displayNetwork) !== "undefined" && !displayNetwork ? 'style="display: none;"' : ''%>>
+							{{#unless displayNetwork}}
```

Comment:
So we stop assuming that these can be not found in the configuration? I'm fine with it because we default to the config file that's used to create the local config at startup time, but I just want to make sure we're all on the same page with that one.